hatchery/hatchery_test.go

```diff
@@ -896,6 +906,10 @@ func Test_TerminateEndpoint(t *testing.T) {
 			return nil
 		}
 
+		getLicenseUserMapsForUser = func(dbconfig *DbConfig, userId string) (*[]Gen3LicenseUserMap, error) {
+			return testcase.mockGen3LicenseUserMaps, nil
+		}
```

Comment:
since you're not testing licenses in this test, i think you could just return `&[]Gen3LicenseUserMap{}` here and remove it from the `testCases`